Examples/test-suite/common.mk

```diff
@@ -706,8 +706,9 @@ CPP20_TEST_BROKEN = \
 # Doxygen comment translation (currently Python and Java) and only if not
 # disabled by configure via SKIP_DOXYGEN_TEST_CASES.
 ifneq ($(SKIP_DOXYGEN_TEST_CASES),1)
-python_HAS_DOXYGEN := 1
+csharp_HAS_DOXYGEN := 1
 java_HAS_DOXYGEN := 1
+python_HAS_DOXYGEN := 1
 
 $(eval HAS_DOXYGEN := $($(LANGUAGE)_HAS_DOXYGEN))
```

Comment:
I realise this isn't part of the change proposed here and so would be better addressed separately (and probably after this gets merged to avoid creating merge conflicts), but the `$(eval` ... `)` part of this seems to serve no purpose - there's even an example in the "Computed Names" section of the GNU make manual which is essentially the same as above:

```
a_dirs := dira dirb
1_dirs := dir1 dir2

a_files := filea fileb
1_files := file1 file2

ifeq "$(use_a)" "yes"
a1 := a
else
a1 := 1
endif

ifeq "$(use_dirs)" "yes"
df := dirs
else
df := files
endif

dirs := $($(a1)_$(df))
```

It indeed seems to work without the eval part.  Am I missing something?

Reply:
FYI I've removed the `$eval(`...`)` here in 9a12b0e5bc84126186c4f16c48783f1262554d69 - I checked and git can automatically merge with the changes here, but if for some reason it doesn't for you this is what's going on.